tests/clims/api/endpoints/test_task_definition.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+
+from sentry.testutils import APITestCase
+from clims.plugins.demo.dnaseq import DemoDnaSeqPlugin
+
+
+class TaskDefinitionTest(APITestCase):
+    endpoint = 'clims-api-0-task-definition'
+
+    def setUp(self):
+        self.clean_workflow_engine_state()
+        self.app.plugins.install_plugins(DemoDnaSeqPlugin)
+        self.has_context()
+
+    def test_get(self):
+        from clims.plugins.demo.dnaseq.workflows.sequence import SequenceSimple
+        from clims.plugins.demo.dnaseq.models import ExamplePlate, ExampleSample
+
+        # Install with plugin API
+        sample_count = 2
+        cont = ExamplePlate(name="cont1")
+        for x in range(1, sample_count + 1):
+            sample = ExampleSample(name="sample-{}".format(x))
+            cont.append(sample)
+        cont.save()
+        workflow = SequenceSimple()
+        workflow.assign(cont)
```

Comment:
Yes, but we also (currently at least) support to assign a whole container in the high-level API. The workflow `SequenceSimple` is configured to take each substance from the container and assign it to the workflow. I figured that this might be handy, but maybe it's just confusing.